account_vat_period_end_statement/views/report_vatperiodendstatement.xml

```diff
@@ -0,0 +1,251 @@
+                               <t t-set="total_to_pay" t-value="(total_to_pay - statement.tax_credit_amount)"/>
+                           </tr>
+
+                           <!-- ...Other tot debit/credit -->
+                           <t t-foreach="statement.generic_vat_account_line_ids" t-as="generic_vat">
+                           <tr>
+                               <td class="text-right"><span t-esc="generic_vat.name"/></td>
+                               <td class="text-right"><span t-esc="generic_vat.account_id.name"/></td>
+                               <td class="text-right"><span t-esc="formatLang(env, generic_vat.amount*-1)"/></td>
+                               <!-- sum -->
+                               <t t-set="total_to_pay" t-value="(total_to_pay + (generic_vat.amount*-1))"/>
+                           </tr>
+                           </t>
+
+                           <!-- Total to pay -->
+                           <tr>
+                               <span t-if="(total_to_pay >= 0)">
+                                   <td/>
+                                   <td class="text-right"><strong>Total To Pay</strong></td>
+                                   <td class="text-right"><strong><span t-esc="formatLang(env, total_to_pay)"/></strong></td>
+                               </span>
+
+                               <span t-if="not (total_to_pay >= 0)">
+                                   <td/>
+                                   <td class="text-right"><strong>Total Credit</strong></td>
+                                   <td class="text-right"><strong><span t-esc="formatLang(env, (total_to_pay * -1))"/></strong></td>
+                               </span>
+                           </tr>
+                    </table>
+                    <!-- payment info -->
+                    <t t-if="statement.payment_ids">
+                        <br/>
+
+                    <table class="table table-condensed">
+                            <t t-foreach="statement.payment_ids" t-as="payment_line">
+                                <tr>
+                                    <!--td t-esc="Importo versato (Estremi del versamento: data ' + payment_line.date + payment_line.journal_id.name)" /-->
+                                    <td t-esc="'Importo versato (Estremi del versamento: data '+ time.strftime('%d/%m/%Y', time.strptime(payment_line.date, '%Y-%m-%d')) + ' - ' +payment_line.journal_id.name+')'" />
+                                    <td class="text-right" t-esc="formatLang(env, payment_line.debit)" />
+                               </tr>
+                               <tr>
+                                   <td colspan="2">
+                                   Reverse charge: comprende RevCharge, Op.SM, Op.Vat., Op.FUE, Beni UE in Italia, Beni FUE in Italia, Cell., Microproc.
+    Operazioni UE: Servizi Si Intra, Servizi No Intra, Beni UE, Rev.Charge UE, Cell. UE, Microproc. UE
```

Comment:
Questa compare già nel .pot, andrebbe tradotta.

Reply:
@primes2h lo farei, ma non sono sicuro di cosa tradurre: ho il sospetto che alcune diciture debbano rimanere in italiano ,ad esempio `Beni UE in Italia`.
Se mi indichi cosa tradurre procedo

Reply:
Se sai indicarmi da dove provengono (esiste un elenco dal quale sono stati presi?) oppure mi espliciti tutti i termini abbreviati con il punto posso darti una mano.

Reply:
@primes2h purtroppo non ho grandi conoscenze della parte contabilità quindi non so cosa siano né da dove siano stati presi questi termini.
Ho trovato che il [commit che li introduce](https://github.com/OCA/l10n-italy/commit/f3ded280c0c596c54a73a7b016793b07967209eb) è stato fatto da @DAMilani e @labaggio, quindi magari loro ne sanno qualcosa.
Forse anche @eLBati potrebbe saperne qualcosa più di noi.

Reply:
È la prima volta che li vedo

Reply:
Allora @primes2h aspetterei ancora qualche giorno per vedere se qualcuno con più conoscenze contabili si fa vivo (magari @stevech091?), altrimenti la porterei avanti anche così.
Non credo sia un problema bloccante, che ne pensi?

Reply:
No, non è bloccante. È che non mi sembrano chiarissimi nemmeno in italiano. :-)

Es.alcuni termini che non capisco sono:
Op.SM
Op.Vat.
FUE (Fuori UE?)

A sto punto aspettiamo giusto qualche giorno sperando che qualche buon'anima si faccia sentire.
Estendo la richiesta a @OCA/local-italy-developers

Reply:
> @primes2h purtroppo non ho grandi conoscenze della parte contabilità quindi non so cosa siano né da dove siano stati presi questi termini.
> Ho trovato che il [commit che li introduce](https://github.com/OCA/l10n-italy/commit/f3ded280c0c596c54a73a7b016793b07967209eb) è stato fatto da @DAMilani e @labaggio, quindi magari loro ne sanno qualcosa.
> Forse anche @eLBati potrebbe saperne qualcosa più di noi.

La mia modifica riguarda alcune traduzione in italiano, ma i  termini che indicate sono oscuri anche a me..